core/tests/webdriverio_utils/ContributorDashboardPage.js

```diff
@@ -52,6 +56,18 @@ var ContributorDashboardPage = function() {
     '.e2e-test-question-suggestion-review-modal-header');
   var usernameContainer = $('.e2e-test-username');
   var reviewRightsDiv = $('.e2e-test-review-rights');
+  var selectorContainer = $('.e2e-test-language-selector');
+
+  var _openLanguageSelector = async function() {
+    await action.click('Test Language Selector Container', selectorContainer);
```

Comment:
WHy are we having `Test` prefix here?

Reply:
I wrote it in the sense of Unit tests and forgot it was e2e tests. Sorry for that. Updated/